llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -1451,6 +1451,21 @@ enum NodeType {
   VECREDUCE_UMAX,
   VECREDUCE_UMIN,
 
+  // Partial Reduction nodes. These represent multiply-add instructions because
+  // Input1 and Input2 are multiplied together first. This result is then
```

Comment:
I thought the behaviour of this node is that Input1 and Input2 are first sign- (for SMLA) or zero- (for UMLA) extended to be the same element type as Acc.

nit: `Input1` and `Input2` are not defined yet. What about starting with a description of the node with operands, e.g. `PARTIAL_REDUCE_*MLA(Acc, Input1, Input2)` ?